.circleci/config.yml

```diff
@@ -67,7 +69,66 @@ workflows:
             branches:
               only:
                 - main
-                - whitelabel
+  backend-production:
+    jobs:
+      - backend-build:
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              ignore: /.*/
+      - administration-build:
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              ignore: /.*/
+      - martin-build:
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              ignore: /.*/
+      - backend-pack:
+          requires:
+            - backend-build
+          filters:
+            tags:
+              only: /.*/
+      - backend-health-check:
+          requires:
+            - backend-pack
+      - administration-pack:
+          requires:
+            - administration-build
+          filters:
+            tags:
+              only: /.*/
+      - martin-pack:
+          requires:
+            - martin-build
+          filters:
+            tags:
+              only: /.*/
+      - meta-pack:
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              ignore: /.*/
+      - deploy:
+          name: deploy-production
+          server: entitlementcard.app
+          context:
+            - entitlementcard-apt
+          requires:
+            - backend-pack
+            - administration-pack
+            - martin-pack
+            - meta-pack
+          filters:
+            tags:
+              only: /.*/
```

Comment:
I'm not a CircleCI expert, but maybe instead of duplicating the workflow just because we want to run one additional job of it for tags, we could maybe use a single `backend` workflow (as before), set the filters of each job to
```yaml
          filters:
            tags:
              only: /.*/
```
(which should make them run on every branch and on every tag)
and have two jobs of the kind `deploy`:
```yaml
      - deploy:
          name: deploy-staging
          server: staging.entitlementcard.app
          context:
            - entitlementcard-apt
          requires:
            - backend-pack
            - backend-health-check
            - administration-pack
            - martin-pack
            - meta-pack
          filters:
            branches:
              only:
                - main
      - deploy:
          name: deploy-production
          server: entitlementcard.app
          context:
            - entitlementcard-apt
          requires:
            - backend-pack
            - administration-pack
            - martin-pack
            - meta-pack
          filters:
            branches:
              ignore:
                - /.*/
            tags:
              only: /.*/
```

Again, I'm not sure if that works, but that was my idea when reading https://circleci.com/docs/workflows/#executing-workflows-for-a-git-tag

Reply:
no you are probably right. that makes sense. More expert than me :)